templates/config.edn

```diff
@@ -205,78 +233,86 @@
     :inputs [:today :7d-after]
     :collapsed? false}]}
 
- ;; Add your own commands to slash menu to speedup.
- ;; E.g.
+ ;; Add custom commands to the command palette
+ ;; Example usage:
  ;; :commands
  ;; [
- ;; ["js" "Javascript"]
- ;; ["md" "Markdown"]
- ;; ]
- :commands
- []
-
- ;; By default, a block can only be collapsed if it has some children.
- ;; `:outliner/block-title-collapse-enabled? true` enables a block with a title
- ;; (multiple lines) can be collapsed too. For example:
+ ;;  ["js" "Javascript"]
+ ;;  ["md" "Markdown"]
+ ;  ]
+ :commands []
```

Comment:
I added this example and it doesn't show anything new in my command palette (cmd-shift-p) or to my slash commands (as the previous comment referenced) This needs much better documentation/explanation as to what it does and how to use it.